src/test/kotlin/sirius/kernel/commons/StringsTest.kt

```diff
@@ -112,6 +112,10 @@ class StringsTest {
         assertTrue(Strings.isHttpUrl("HTTPS://example.com"))
         assertTrue(Strings.isHttpUrl("http://example.com"))
         assertTrue(Strings.isHttpUrl("Http://example.com?foo=bar"))
+        assertTrue(Strings.isHttpUrl("http://example.com?user=foo&password=bar"))
+        assertTrue(Strings.isHttpUrl("http://example.com?user=foo&password="))
```

Comment:
We can leave them, but I meant something like `http://user@server.com/path` or `http://user:password@server.com/path`